src/ripple/overlay/impl/OverlayImpl.cpp

```diff
@@ -912,6 +914,61 @@ OverlayImpl::crawl()
     return jv;
 }
 
+Json::Value
+OverlayImpl::getLocalServerInfo()
+{
+    Json::Value jv;
+
+    bool const humanReadable = false;
+    bool const admin = false;
+    bool const counters = false;
+
+    Json::Value server_info = app_.getOPs().getServerInfo(humanReadable, admin, counters);
+
+    // Filter out some information
+    server_info.removeMember(jss::hostid);
+    server_info.removeMember(jss::load_factor_fee_escalation);
+    server_info.removeMember(jss::load_factor_fee_queue);
+
+    if (server_info.isMember(jss::validated_ledger))
+    {
+        Json::Value& validated_ledger = server_info[jss::validated_ledger];
+
+        validated_ledger.removeMember(jss::base_fee);
+        validated_ledger.removeMember(jss::reserve_base_xrp);
+        validated_ledger.removeMember(jss::reserve_inc_xrp);
+    }
+
+    jv[jss::info] = std::move(server_info);
```

Comment:
We move from both `server_info` and `validator_sites` in this function, and they shouldn't be used after they are used from (they aren't, but I'd like to make that clear from the code for easy inspection and to make futures changes less likely to use them). I'd either:
1) Just put them into their own scopes, starting at the declaration of the varialbe and ending when they are moved.
or
2) Use a lambda to set the values. I.e.: `jv[jss::info] = [&]{...}()`. RVO should make this as efficient as moving.
Either way is fine with me.

Reply:
This is now pulled into separate functions (so they can be controlled through the config file), so no moving necessary.